ReactiveCocoa.xcodeproj/project.pbxproj

```diff
@@ -1961,6 +2357,211 @@
 /* End PBXTargetDependency section */
 
 /* Begin XCBuildConfiguration section */
+		A9B315591B3940610001CB9C /* Debug */ = {
+			isa = XCBuildConfiguration;
+			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
```

Comment:
Can you bump the xcconfigs dependency to include jspahrsummers/xcconfigs#40 which adds watchOS support. You can then apply those configurations and delete almost all of these. Take a look at the iOS targets for what needs to remain.